plugins/statsd/statsd_input.go

```diff
@@ -125,20 +125,24 @@ func (s *StatsdInput) Stop() {
 // Parses received raw statsd bytes data and converts it into a StatPacket
 // object that can be passed to the StatMonitor.
 func (s *StatsdInput) handleMessage(message []byte) {
-	stats, err := parseMessage(message)
+	stats, warn, err := parseMessage(message)
 	if err != nil {
 		s.ir.LogError(fmt.Errorf("Can not parse message: %s", message))
 		return
 	}
 
+	if warn != nil || len(warn) > 0 {
+		s.ir.LogError(fmt.Errorf("Warning Part of message could not be parsed: %s", warn))
+	}
+
 	for _, stat := range stats {
 		if !s.statAccum.DropStat(stat) {
-			s.ir.LogError(fmt.Errorf("Undelivered stat: %v", stat))
+			s.ir.LogError(fmt.Errorf("Undelivered stat: %s", stat))
```

Comment:
%v will output the stat values, converting to string where necessary, whereas %s will generate an error for non-string attributes of the struct. If we're going to change it at all, using %+v to include the field names is a better choice.